### PR TITLE
add reference ui

### PR DESCRIPTION
**Pull request by sumana10**

### CODE REVIEW

File: week-10/userApi/.vite/deps/_metadata.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
```
Comment (Praashh) [inline week-10/userApi/.vite/deps/_metadata.json:1]: don't think, we need this .vite configuration, can you tell me why we have this?

**Review by Praashh — COMMENTED** [review]
can we add a component folder to make it easy to understand the structure of the app for beginners ?

**Comment by sumana10** [comment]
> image
> 
> can't see the image in the README, is this working for you?

https://github.com/sumana10/assignments-cohort3/tree/solution-branch/week-10/authSystem

**Comment by sumana10** [comment]
> can we add a component folder to make it easy to understand the structure of the app for beginners ?

This is pretty basic for anyone who has been starting out for a week or so. There is nothing complex.

**Review by Praashh — APPROVED** [review]
got it, merging this, Thank you!!

**Pull request merged**